ui/imports/utils/Global.qml

```diff
@@ -72,7 +71,13 @@ QtObject {
     signal setNthEnabledSectionActive(int nthSection)
     signal appSectionBySectionTypeChanged(int sectionType, int subsection, int subSubsection, var data)
 
-    signal openSendModal(string address)
+    signal openSendModal(int sendType, string senderAddress,
```

Comment:
... that said, this should/could be a simple function instead, with defaulted parameters, and then you can just re-emit a similar signal with all the parameters, instead of having to fill all the params to this signal here